conbench/entities/benchmark_result.py

```diff
@@ -570,7 +571,7 @@ def ui_mean_and_uncertainty(values: List[float], unit: str):
 
     if len(values) < 3:
         # Show each sample with five significant figures.
-        return "; ".join(str(sigfig.round(v, sigfigs=5)) for v in values)
+        return "; ".join(f"{numstr(v, sigfigs=5)} {unit}" for v in values)
```

Comment:
This line here adds the unit, i.e. this takes care of "data column: show unit when there are only one or two values" tracked in #1154.